components/org.wso2.carbon.identity.keyrotation/src/main/java/org.wso2.carbon.identity.keyrotation/config/KeyRotationConfigValidator.java

```diff
@@ -0,0 +1,50 @@
+package org.wso2.carbon.identity.keyrotation.config;
+
+import org.apache.commons.lang3.BooleanUtils;
+import org.wso2.carbon.identity.keyrotation.util.KeyRotationException;
+
+import java.io.File;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+/**
+ * Validator for configuration properties retrieved by the {@link KeyRotationConfigProvider}.
+ */
+public class KeyRotationConfigValidator {
+
+    private static final KeyRotationConfigValidator instance = new KeyRotationConfigValidator();
+
+    public static KeyRotationConfigValidator getInstance() {
+
+        return instance;
+    }
+
+    public void validateURI(String name, String value) throws KeyRotationException {
```

Comment:
Let's add method comments to public methods. Check in other places as well

Reply:
Addressed with https://github.com/wso2/identity-tools/pull/22/commits/0151a748192960c182603b555329eb3f1e0229ea.